### ISSUE TITLE
Configuration via mounted volumes

### ISSUE THREAD

**Issue by felixSchl** [SumoLogic/sumologic-collector-docker#22]
One way to configure sumologic from outside of the container and without baking a custom image would be to share part of the host's file system with the container. However, it seems that the key configuration files to sumologic are inconveniently placed for this usecase. For example the `sumo.conf` is expected to sit in `/etc/sumo.conf`. That would mean that I need to mount a volume `/etc` in the container. This leaves me with two options: expose the entire host's `/etc` directory via `/etc -> /etc` or keep my file somewhere else: `/etc/foo -> /etc`. The problem with the latter is that now `/etc` may be missing other files required by the image since `/etc` not points to `/etc/foo` on the host... The same issue holds true for the `user.properties` file which is expected to live in the sumologic installer directory.

**Comment by duchatran** [comment]
I think most of not all properties in these files can be passed to collector now (https://help.sumologic.com/Send_Data/Installed_Collectors/Step_4._Install_the_Collector/02_Quiet_Mode_Installation_Method). Does this solve your problem?

**Comment by felixSchl** [comment]
You mean configuring the collector during installation? I have no control over that, neither as a direct consumer of the image nor in a downstream image, since installation happens here: https://github.com/SumoLogic/sumologic-collector-docker/blob/master/Dockerfile#L12. I can only imagine to provide symlinks to these files in a more convenient location in a downstream image, but I was hoping for a way not to have to create a downstream image at all.

What I am thinking is this:

```
RUN mkdir -p /etc/sumo && mv /etc/sumo{.conf,} && ln -s /etc/{sumo/,}sumo.conf
```

... and something similar for the install location of the collector.

**Comment by stevezau** [comment]
@felixSchl my PR just got merged #27 which may help with your question. (guessing).

**Comment by duchatran** [comment]
@felixSchl : closing this issue, feel free to reopen if @stevezau's latest change doesn't help
